bundles/org.openhab.binding.fritzboxtr064/README.md

```diff
@@ -0,0 +1,103 @@
+# TR-064 Binding
+
+This binding brings support for internet gateway devices that support the TR-064 protocol.
+It can be used to gather information from the device and/or re-configure it.
+
+Tested devices are:
+- AVM Fritzbox 6490 Cable
+
+## Supported Things
+
+Three things are supported:
+
+- `rootDevice`: the internet gateway device itself
+- `subDevice`: a sub-device of a `rootDevice` (e.g. a WAN interface) 
+- `subDeviceLan`: a special type of sub-device that supports MAC-detection
+
+## Discovery
+
+The root device needs to be added manually, after that sub-devices are detected automatically.
+It is highly recommended to configure that way and NOT rely on manual configuration.
+
+## Thing Configuration
+
+All things have a `refresh``parameter.
+It sets the refresh-interval in seconds for each device channel.
+The default value is 60.
+
+### `rootdevice`
+
+The `host` parameter is needed to communicate with the device.
+It can be a hostname or an IP address and defaults to `fritz.box`.
```

Comment:
The binding needs it, but the user does not have to configure it. In 99+% of all cases the default `fritz.box` will be the correct setting. Maybe the sentence before should be reworded.

Reply:
Once we have a dedicated thing type for fritz boxes, I am ok with using this default for it (but not for the generic `cpe`). For the fritzbox thing type, the parameter must not be "mandatory", though, but only optional (since there is a default).

Note though that I have seen many networks already, where the resolution of `fritz.box` did not work and one had to resort to using the ip address, so I would not agree that it'll work in 99+% of the cases.